src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -163,18 +164,43 @@ export function CreateExportModal({
             {" "}
             {issuesToExport.length}
           </Text>
-          <Button
-            ml="auto"
-            size="sm"
-            onClick={() => {
-              exportIssues(
-                issuesToExport,
-                issueStatus.filter((s) => includedIssueStatus.includes(s.name)),
-              );
-            }}
+          <Tooltip
+            withArrow
+            multiline
+            w={200}
+            fz={14}
+            fw={500}
+            openDelay={200}
+            closeDelay={200}
+            opened={
+              (endDate === null
+                || startDate === null
+                || issuesToExport.length === 0)
+              && hoveredoverexport
+            }
+            ta="center"
+            color="primaryBlue"
+            variant="filled"
+            label="Please select Issue Types and Status with a date range with at least 1 issue to export"
```

Comment:
```suggestion
            label="No issues are exportable with this configuration"
```
A bit more straight-forward and valid for a different configuration style.